scripts/common/preflights.sh

```diff
@@ -365,14 +365,10 @@ function host_preflights() {
         # TODO: report preflight fail
     else
         # interactive terminal
-        if [ -t 0 ] ; then
+        if canPrompt; then      
             if ! "${DIR}"/bin/kurl host preflight "${MERGED_YAML_SPEC}" ${opts} </dev/tty ; then
-                printf "${RED}Host preflights have failures. Do you want to proceed anyway? ${NC} "
-                if ! confirmN "-t 30"; then
-                    report_install_fail "preflight"
-                    bail "Use the \"preflight-ignore\" flag to proceed."
-                fi
-                # TODO: report preflight fail
```

Comment:
`kurl` already prompts the user here, although the default is backward from what we wanted in the script (Yes is default in the binary)

Reply:
but what if there are failures?

Reply:
Yes, I failed to consider that case. You'll get a double prompt if you hit No to exit on the warnings. I think the solution could be using different exits codes in `kurl`. I looked at this for a second and might need some help dealing with the errors package if we want to go that route. I reverted it for now.